about.html

```diff
@@ -33,7 +33,7 @@
                 <span class="bar"></span>
             </a>
     
-            <a href="/index.html"><img class="main-page-logo" src="img/smartbuild.jpg" alt=""></a>
+            <a href="/index.html"><img class="main-page-logo" src="img/smartbuild.jpg" alt="https://smartbuildsystems.com/wp-content/uploads/2018/01/cropped-SmartBuild-Logo-768x388.png"></a>
```

Comment:
- [ ] You did a great job on putting a text as requested by the previous reviewer but I highly recommend you put the meaningful text according to the image, not the link.

  _**Hint:**_   `<img class="main-page-logo" src="img/smartbuild.jpg" alt="Logo">`

### Make to change every `alt` text in all HTML files.